cmd/ctr-remote/commands/convert.go

```diff
@@ -131,7 +130,7 @@ When '--all-platforms' is given all images in a manifest list must be available.
 			}
 			layerConvertFunc = estargzconvert.LayerConvertFunc(esgzOpts...)
 			if !context.Bool("oci") {
-				logrus.Warn("option --estargz should be used in conjunction with --oci")
+				return errors.New("option --estargz should be used in conjunction with --oci")
```

Comment:
Not a hard error

Reply:
hmm. But the snapshotter needs that annotations 😂. without the annotations, rpull will fetch all the contents

Reply:
thanks! Since it is not issue, I am close it.